Android/res/values-es/strings.xml

```diff
@@ -0,0 +1,454 @@
+    <string name="pref_connection_actions">Mientras este conectado al drone</string>
+    <string name="pref_connection_actions_summary">Acciones a realizar mientras está conectado al Drone\nPara aplicarlas desconecte y vuelva a conectar</string>
+    <string name="pref_connection_type">Tipo de conexion de telemetria</string>
+    <string name="pref_connection_type_summary">¿Qué vínculo a utilizar para conectarse con el drone?</string>
+    <string name="pref_dialog_entry_always">Siempre</string>
+    <string name="pref_dialog_entry_ask">Preguntar</string>
+    <string name="pref_dialog_entry_never">Nunca</string>
+    <string name="pref_dialog_selection_reset_desc"><![CDATA[Resetear seleccion a traves de: Opciones -> Interfaz de usuario -> Preferencias de Dialogos]]></string>
+    <string name="pref_dialogs_title">Preferencias de dialogos</string>
+    <string name="pref_dronie_creation_message">Por favor alejate 8m antes de empezar con el Dronie</string>
+    <string name="pref_dronie_creation_title">¿Hacer un Dronie?</string>
+    <string name="pref_enable_kill_switch_summary">Si se activa, la opción Kill Switch aparecerá bajo la sección Avanzadas en el menú, lo que permite desarmar el vehículo inmediatamente.</string>
+    <string name="pref_enable_kill_switch_title">Activar el Kill Swith</string>
+    <string name="pref_enable_tts">Activar Texto a Voz</string>
+    <string name="pref_enable_tts_summary">Mensajes audibles para eventos importantes</string>
+    <string name="pref_enable_udp_server_title">Habilitar el Ping UDP al servidor</string>
+    <string name="pref_flight_data">Datos de vuelo</string>
+    <string name="pref_follow_me_mode_enabled">Habilitar el modo FollowMe(Sigueme)</string>
+    <string name="pref_follow_me_mode_enabled_summary">En desarrollo</string>
+    <string name="pref_forget_bluetooth_device_address">Olvidar el dispositivo por defecto</string>
+    <string name="pref_forget_bluetooth_device_address_confirm">¿Estás seguro?</string>
+    <string name="pref_forget_bluetooth_device_title">Olvidar el dispositivo %s</string>
+    <string name="pref_ground_collision_warning_summary">Advertir sobre una inminente colisión con el suelo.</string>
+    <string name="pref_ground_collision_warning_title">Colision con el suelo</string>
+    <string name="pref_guided_mode_on_long_press">Pulsación larga para modo GUIDED(guiado)</string>
+    <string name="pref_guided_mode_on_long_press_summary">Habilitar entrar en modo GUIDED(Guiado) con una pulsación larga</string>
+    <string name="pref_heading_mode">Habilitar el modo FPV dirigido</string>
+    <string name="pref_heading_mode_summary">Cambiar el modo dirigido del fpv de (-180º a 180º)</string>
+    <string name="pref_keep_screen_bright">Mantener la pantalla encendida</string>
+    <string name="pref_keep_screen_bright_summary">Comprobar que la pantalla y el brillo se mantienen ( afecta a la vida de la bateria)</string>
```

Comment:
is the space between parenthesis desired or bug, we should either add it at the end or remove it from the beginning ```( afecta a la vida de la batería )``` or ```(afecta a la vida de la batería)``` also batería has ```í``` an "accent"